### PR TITLE
Picker Vertical Text Alignment

### PR DESCRIPTION
**Pull request by yurkinh**
### Description of Change
 Implements VerticalTextAlignment property for picker
 
 ### Additions made

 * Adds 'VerticalTextAlignment'  to the `ITextAligment`  interface
 * Adds VerticalTextAlignment property map to PickerHandler
 * Adds VerticalTextAlignment methods map to PickerHandler for WinUI,  Android and iOS
 
 ### PR Checklist
 * [x]  Targets the correct branch
 * [x]  Tests are passing (or failures are unrelated)
 * [ ]  Targets a single property for a single control (or intertwined few properties)
 * [ ]  Adds the property to the appropriate interface
 * [ ]  Avoids any changes not essential to the handler property
 * [x]  Adds the mapping to the PropertyMapper in the handler
 * [x]  Adds the mapping method to the Android, iOS, and Standard aspects of the handler
 * [ ]  Implements the actual property updates (usually in extension methods in the Platform section of Core)
 * [x]  Tags ported renderer methods with [PortHandler]
 * [x]  Adds an example of the property to the sample project (MainPage)
 * [ ]  Adds the property to the stub class

 
 #### Does this PR touch anything that might affect accessibility?
 * [ ]  APIs that modify focusability?
 * [ ]  APIs that modify any text property on a control?
 * [ ]  Does this PR modify view nesting or view arrangement in any way?
 * [ ]  Is there the smallest possibility that your PR will change accessibility?
 * [ ]  I'm not sure, please help me
 
 If any of the above checkboxes apply to your PR then the PR will need to provide testing to demonstrate that accessibility still works.

### CODE REVIEW

File: src/Core/tests/DeviceTests/Stubs/SearchBarStub.cs [hunk 1]
```diff
@@ -18,6 +18,8 @@ public partial class SearchBarStub : StubBase, ISearchBar
 
 		public TextAlignment HorizontalTextAlignment { get; set; }
 
+		public TextAlignment VerticalTextAlignment { get; set; }
+
```
Comment (jsuarezruiz) [inline src/Core/tests/DeviceTests/Stubs/SearchBarStub.cs:22]: Could you include a test that verify the new property?

Comment (yurkinh) [reply src/Core/tests/DeviceTests/Stubs/SearchBarStub.cs:22]: Added device tests similar to what we have for HorizontalTextAlignment

**Comment by rmarinho** [comment]
@yurkinh can you rebase?

**Comment by yurkinh** [comment]
@rmarinho Done.

**Comment by rmarinho** [comment]
Thanks

**Review by jsuarezruiz — APPROVED** [review]

**Comment by jsuarezruiz** [comment]
Just waiting to see the Build results, but looks nice.

**Pull request merged**